### DISCUSSION TITLE
It's not over yet, they're still trying to attract signatories:

### DISCUSSION THREAD

**Discussion by cmpunches** [rms-support-letter/rms-support-letter.github.io#4930] [General]
https://twitter.com/mmillions/status/1376882475041964035

They are collecting signatures in secret email threads now.  I'm sure it's "totally a reliable process that we aren't misusing to pad our numbers, we swear".  If they'll lie in their campaign about the material facts, they'll lie about their signatories, too, so, we need to drive to continue as we have been -- if you need sleep, get sleep, and carry on the effort!

We need people to continue to promote this letter and attract signers.  We need more international reach.  We need more corporate reach where possible.

**Comment by nukeop** [comment]
Without transparency the whole process is illegitimate.

**Reply by KOLANICH** [reply]
I should remember that we don't have enough checks. Transparency is must have, but not enough. For example in the harras-letter one can find some signatures explicitly making fun of their lack of verification that are obvious to any person knowing Russian enough: it is simply illegal (in the sense if one would try to change his name and surname in ID to such ones, he would be refused, and one cannot acquire a bith certificate for a child using such words, and one cannot register a company using such words in title) to have such names and surnames and organization names. So they are already laughingstocks. But it is not a reason for this initiative to be such (though it easily can be). Though they are very gender- and orientation-inclusive by allowing such "persons" and "organizations" without any doubt ;)

So I propose to allow signatures only of real devs (users are usually worthless, unless they attract money or other benefits, so their signature usually has much less weight, free software can exist without pure-users, but without the ones who has created the software it cannot exist) and rigorously (not in a sense of govt ID, but in the sense of contribution - the person must contribute something original to be considered a developer, the criteria were already named) verify them.
2. The signatures of persons not satisfying the criteria should also be registered, but separately. They shouldn't be counted when claiming that the petition has some count of signatures - they are much easier to fake. But even for them additional checks are needed, such as if a name looks like a joke on some language or if the message was accepted from a fake email address.

**Reply by KOLANICH** [reply]
`remember` -> `remind`

**Comment by cmpunches** [comment]
> Without transparency the whole process is illegitimate.

@nukeop could not agree more, but, remember they're running a mixed-media viral campaign effort, so, if enough people believe it, it may as well be true.

**Comment by cmpunches** [comment]
At this point we have almost twice as many signatures.  We gotta keep climbing so they don't close the gap.

**Comment by hinell** [comment]
If so, it would be nice to contact signatories to poll them whether they signed or not (assuming signatories are real).

**Comment by cmpunches** [comment]
> If so, it would be nice to contact signatories to poll them whether they signed or not (assuming signatories are real).

@hinell  AFTER their campaign has concluded.  If you start to attack their process during it, they'll just adapt and make it harder to do that with worse long term consequences.

**Comment by cmpunches** [comment]
That said, it's still important to *track* irregularities and process changes as well as their signature counts at those points in the timeline, because that'll be the data needed when that starts to be a thing.

**Comment by cmpunches** [comment]
At the core of this campaign, Molly is a vulnerable narcissist, so, expect unproven victim-stancing, like "harassment of our signatories", which, of course, was their modus operandus towards this campaign from the get-go.  Most of their accusations will end up being confessions.

**Comment by aleferri** [comment]
https://github.com/sticks-stuff/highlight-RMS-supporters

They are going against us.
This is not lawful, so i reported them, but write that to the front page so people will know who is in the right and who is a serial harasser

**Reply by KOLANICH** [reply]
Such tools are double-edged swords. (Ironically, the idiom in Russian is literally sounds as `every stick has 2 ends`. ;) They can be used to highlight foes in order to harrass them , ... or to highlight camrades. Or the mass-banning tool: he can ban us in his orgs, projects and issues, but when he does it, he willfully isolates not only us from himself, but also himself from us, and also makes himself inacceptable as a part of a team of any FOSS project which official position is not acdepting the "open-letter" (let's call it the "harras-letter", this name fits more there). We need tools to detect the ones who actually used such mass-blocking tools in order to demand either their resignation from orgs, not on behalf of their opinions, but on behalf of refusal to do their functions in an org (i.e. ones taking large part in developing an official package manager for a certain programming language must allow everyone to review and comment their proposals, and in the case of a mass-ban it is equivalent to the person doing mass ban deciding "my proposals must not be reviewed by ...", but when contributing to a large project, and not own project, it is not the decision that is up to him). The ideal solution is to remove the restriction on commenting in the case of a personal ban from GH, it has no uses other than harrasment.

**Reply by Bluesboy** [reply]
https://youtu.be/kxPWwX-Apw4?t=3410 
I agree with what Eli said about this 'black list'

**Comment by hinell** [comment]
@aleferri I've already raised an issue here: #4910 , it was closed . You can take a report message from out there and use it to report the profile and repository.

@KOLANICH Whatever. It's very compromising to use such tools to go after foes. Just as dumb as it could be.

**Reply by aleferri** [reply]
I did, i received a response. They are investigating on the user and the repository.

**Reply by hinell** [reply]
Same here.
